test/JDBC/expected/BABEL-5608-vu-prepare.out

```diff
@@ -0,0 +1,238 @@
+GO
+~~START~~
+text
+CHECK (((babel_5608_vu_prepare_t4_1_col1)::text = 'ABC'::text))
+~~END~~
+
+
+CREATE TABLE BABEL_5608_vu_prepare_t4_2(BABEL_5608_vu_prepare_t4_2_col1 varchar(max), CONSTRAINT BABEL_5608_vu_prepare_t4_2_c1 CHECK (BABEL_5608_vu_prepare_t4_2_col1 LIKE 'a%'));
+GO
+
+SELECT pg_get_constraintdef(oid) FROM pg_constraint WHERE conrelid=object_id('BABEL_5608_vu_prepare_t4_2');
+GO
+~~START~~
+text
+CHECK (((((babel_5608_vu_prepare_t4_2_col1)::text COLLATE bbf_unicode_cp1_cs_as) ~~* 'a%'::text COLLATE bbf_unicode_cp1_cs_as) AND (((babel_5608_vu_prepare_t4_2_col1)::text >= 'a'::text) AND ((babel_5608_vu_prepare_t4_2_col1)::text < ('a'::text || '?'::text)))))
+~~END~~
+
+
+-- CASE 5: T_ReLabelType(T_Var) LIKE T_CollateExpr(T_Const)
+CREATE TABLE BABEL_5608_vu_prepare_t5_1(BABEL_5608_vu_prepare_t5_1_col1 varchar(max), CONSTRAINT BABEL_5608_vu_prepare_t5_1_c1 CHECK (BABEL_5608_vu_prepare_t5_1_col1 LIKE 'ABC' COLLATE Latin1_General_CI_AI));
+GO
+
+SELECT pg_get_constraintdef(oid) FROM pg_constraint WHERE conrelid=object_id('BABEL_5608_vu_prepare_t5_1');
+GO
+~~START~~
+text
+CHECK ((((remove_accents_internal((babel_5608_vu_prepare_t5_1_col1)::text))::text COLLATE bbf_unicode_cp1_cs_as) ~~* ((remove_accents_internal(('ABC'::text COLLATE "default" COLLATE bbf_unicode_cp1_ci_ai)))::text COLLATE bbf_unicode_cp1_cs_as)))
+~~END~~
+
+
+CREATE TABLE BABEL_5608_vu_prepare_t5_2(BABEL_5608_vu_prepare_t5_2_col1 varchar(max), CONSTRAINT BABEL_5608_vu_prepare_t5_2_c1 CHECK (BABEL_5608_vu_prepare_t5_2_col1 LIKE 'a%' COLLATE Latin1_General_CI_AI));
+GO
+
+SELECT pg_get_constraintdef(oid) FROM pg_constraint WHERE conrelid=object_id('BABEL_5608_vu_prepare_t5_2');
+GO
+~~START~~
+text
+CHECK ((((remove_accents_internal((babel_5608_vu_prepare_t5_2_col1)::text))::text COLLATE bbf_unicode_cp1_cs_as) ~~* ((remove_accents_internal(('a%'::text COLLATE "default" COLLATE bbf_unicode_cp1_ci_ai)))::text COLLATE bbf_unicode_cp1_cs_as)))
+~~END~~
+
+
+-- CASE 6: T_ReLabelType(T_Var) LIKE T_ReLabelType(T_Var)
+CREATE TABLE BABEL_5608_vu_prepare_t6(BABEL_5608_vu_prepare_t6_col1 varchar(max), BABEL_5608_vu_prepare_t6_col2 varchar(max), CONSTRAINT BABEL_5608_vu_prepare_t6_c1 CHECK (BABEL_5608_vu_prepare_t6_col1 LIKE BABEL_5608_vu_prepare_t6_col2));
+GO
+
+SELECT pg_get_constraintdef(oid) FROM pg_constraint WHERE conrelid=object_id('BABEL_5608_vu_prepare_t6');
+GO
+~~START~~
+text
+CHECK ((((babel_5608_vu_prepare_t6_col1)::text COLLATE bbf_unicode_cp1_cs_as) ~~* ((babel_5608_vu_prepare_t6_col2)::text COLLATE bbf_unicode_cp1_cs_as)))
+~~END~~
+
+
+-- CASE 7: T_CollateExpr(T_ReLabel(T_Var)) LIKE T_Const
+CREATE TABLE BABEL_5608_vu_prepare_t7_1(BABEL_5608_vu_prepare_t7_1_col1 varchar(max), CONSTRAINT BABEL_5608_vu_prepare_t7_1_c1 CHECK (BABEL_5608_vu_prepare_t7_1_col1 COLLATE Latin1_General_CI_AI LIKE 'abc'));
+GO
+
+SELECT pg_get_constraintdef(oid) FROM pg_constraint WHERE conrelid=object_id('BABEL_5608_vu_prepare_t7_1');
```

Comment:
I think this should be part of verify script as well. it will ensure that we have expected constraint before and after upgrade

Reply:
Updated in latest rev